dependencies.py

```diff
@@ -31,21 +31,19 @@ def extra_requirements() -> typing.Dict[str, typing.List[str]]:
     #     - We have a copy of these in extras-requirements.txt. If you modify these, make sure to change it
     #       there as well
     extras_require = {
-        # from 1.17.107 boto3 requires botocore>=1.20.107,<1.21.0 which
-        # conflicts with s3fs 2021.8.1 that has aiobotocore~=1.4.0
-        # which so far (1.4.1) has botocore>=1.20.106,<1.20.107
-        # boto3 1.17.106 has botocore>=1.20.106,<1.21.0, so we must add botocore explicitly
+        # aiobotocore 2.4.2 is the latest supported by python-3.7
```

Comment:
```suggestion
        # aiobotocore 2.4.2 is the last version to support Python 3.7
```
Kinda nitpicky, but it's not up to python 3.7 to support aiobotocore... Rather it's the other way around.